backend/python/app/services/implementations/story_service.py

```diff
@@ -125,23 +125,28 @@ def get_story_translations(self, user_id, translator, language, level):
 
     def get_story_translation(self, id):
         try:
-            return (
+            translation = StoryTranslation.query.get(id)
+
+            story_details = (
                 db.session.query(
                     Story.id.label("story_id"),
                     Story.title.label("title"),
                     Story.description.label("description"),
                     Story.youtube_link.label("youtube_link"),
                     Story.level.label("level"),
-                    StoryTranslation.id.label("story_translation_id"),
-                    StoryTranslation.language.label("language"),
-                    StoryTranslation.stage.label("stage"),
-                    StoryTranslation.translator_id.label("translator_id"),
-                    StoryTranslation.reviewer_id.label("reviewer_id"),
                 )
                 .join(StoryTranslation, Story.id == StoryTranslation.story_id)
                 .filter(StoryTranslation.id == id)
                 .one()
             )
+
+            response = {
```

Comment:
nice pattern! clean. 
Ideally, we would fetch all this info in 1 SQL call rather than 2, but I'm happy with this for now to unblock. Can you leave a TODO comment for reducing function to 1 SQL call for the future?